tests/tracking-tests.yml

```diff
@@ -0,0 +1,362 @@
+!TestConfig
+clip_dir: tests/test-clips
+recording_tests:
+- !TestRecording
+  device: Wallaby
+  device_id: 956
+  filename: 551732.cptv
+  group: ORC
+  group_id: 160
+  rec_id: 551732
+  tracks:
+  - !TestTrack
+    confidence: 0.85
```

Comment:
Is start and end frame enough or do we need the full tracks?  (In which case we would definitely need some way of seeing the different tracking and and automatic way of diffing to see whether changes are significant or insignificant.